test/integration/beta_cluster/beta_cluster_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package beta_cluster
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/golden"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBetaCluster(t *testing.T) {
+	gke := tft.NewTFBlueprintTest(t)
+
+	gke.DefineVerify(func(assert *assert.Assertions) {
+		gke.DefaultVerify(assert)
+
+		projectID := gke.GetStringOutput("project_id")
+		location := gke.GetStringOutput("location")
+		clusterName := gke.GetStringOutput("cluster_name")
+		serviceAccount := gke.GetStringOutput("service_account")
+		op := gcloud.Runf(t, "beta container clusters describe %s --zone %s --project %s", clusterName, location, projectID)
+		// save output as goldenfile
+		g := golden.NewOrUpdate(t, op.String(),
+			golden.WithSanitizer(golden.StringSanitizer(serviceAccount, "SERVICE_ACCOUNT")),
```

Comment:
Since this is likely to be the common pattern, we should consider adding `golden.WithStringSanitizer` to reduce this boilerplate.